packages/ui/src/exports/shared/index.ts

```diff
@@ -2,6 +2,7 @@
 export { filterFields } from '../../elements/TableColumns/filterFields.js'
 export { getInitialColumns } from '../../elements/TableColumns/getInitialColumns.js'
 export { Translation } from '../../elements/Translation/index.js'
+export { formatOptions } from '../../elements/WhereBuilder/Condition/Select/formatOptions.js'
```

Comment:
I don't think we should export this. Its a pretty simple function that's we be forced into maintaining. It's also very specific to React Select although the function name is generic.